bin/learn-test

```diff
@@ -57,6 +57,10 @@ OptionParser.new do |opts|
     options[:fail_fast] = f
   end
 
+  opts.on("-e", "--example STRING", "Run examples whose full nested names include STRING (may be used more than once)") do |s|
```

Comment:
So while rspec natively supports [multiple uses of `--example`](https://relishapp.com/rspec/rspec-core/v/3-8/docs/command-line/example-option#multiple-applications-of-example-name-option) flags, the implementation here does not I don't think.

Reply:
I've added this test

```ruby
    it 'accepts multiple examples' do
      output = `cd ./spec/fixtures/rspec-unit-spec && ./../../../bin/learn-test --local --test --example multiple --example accepts`

      expect(output).to include('2 examples, 0 failures')
      expect(output).to_not include('3 examples')
      expect(output).to_not include('1 example')
    end    
```

and updated the fixture:

```ruby
  it 'has multiple specs' do
    expect(subject).not_to be_an_instance_of(String)
  end

  it 'has accepts more than one example' do
    expect(subject).not_to be_an_instance_of(String)
  end  
```

to confirm and it fails. looking into how to glob multiple `--examples` through `OptionParser`